### PR TITLE
Allow set default domain on login

### PR DESCRIPTION
**Pull request by carabasdaniel**

### CODE REVIEW

File: cmd/policy/main.go [hunk 1]
```diff
@@ -92,7 +92,7 @@ type Globals struct {
 var g *Globals
 
 var PolicyCLI struct {
-	Config    string `short:"c" type:"path" help:"Path to the policy CLI config file." default:"${userHome}/.config/policy/config.yaml"`
+	Config    string `short:"c" type:"path" help:"Path to the policy CLI config file." default:"${userHome}/.config/policy/config.json"`
```
Comment (gimmyxd) [inline cmd/policy/main.go:95]: Why are we changing form yaml to json?

File: pkg/cc/config/config.go [hunk 2]
```diff
@@ -180,6 +181,41 @@ func (c *Config) SaveCreds() error {
...
+		DefaultDomain string        `json:"default_domain"`
+		Logging       logger.Config `json:"logging"`
+		CA            []string      `json:"ca"`
+		Insecure      bool          `json:"insecure"`
+	}
+	tmp.FileStoreRoot = c.FileStoreRoot
+	tmp.DefaultDomain = c.DefaultDomain
+	tmp.Logging = c.Logging
+	tmp.CA = c.CA
+	tmp.Insecure = c.Insecure
```
Comment (gimmyxd) [inline pkg/cc/config/config.go:197]: Do we need to write default values to the file, for the settings not provided by the user?

**Review by gimmyxd — APPROVED** [review]

File: cmd/policy/login.go [hunk 3]
```diff
@@ -74,3 +82,33 @@ func (c *LoginCmd) Run(g *Globals) error {
 	<-g.App.Context.Done()
 	return nil
 }
+
+func checkDefault(g *Globals, c *LoginCmd) (bool, error) {
+	setDefault := false
```
Comment (oanatmaria) [inline cmd/policy/login.go:87]: if you set here `setDefault := c.DefaultDomain` you won't need the lines 98-100

File: cmd/policy/login.go [hunk 4]
```diff
@@ -74,3 +82,33 @@ func (c *LoginCmd) Run(g *Globals) error {
...
+			"n":   false,
+			"Y":   true,
+			"N":   false,
+		}).Do()
+	}
+	if c.DefaultDomain {
+		setDefault = true
+	}
+
+	// Reset all defaults to false to return true and set the current login as default
```
Comment (oanatmaria) [inline cmd/policy/login.go:102]: // Reset all defaults to false to return true and set the current login as default
```suggestion
	// Reset all defaults to false to return true in order to set the current login as default
```

**Review by oanatmaria — APPROVED** [review]

**Pull request merged**